WORKING_GROUPS.md

```diff
@@ -0,0 +1,239 @@
+# OpenJS Core Working Groups
+
+OpenJS Core Working Groups (WG) are autonomous projects created by the
+[Cross Project Council (CPC)][].
+
+Working Groups can be formed at any time but must be ratified by the CPC.
+Once formed the work defined in the Working Group charter is the
+responsibility of the WG rather than the CPC.
+
+It is important that Working Groups are not formed pre-maturely. Working
+Groups are not formed to *begin* a set of tasks but instead are formed
+once that work is already underway and the contributors
+think it would benefit from being done as an autonomous project.
+
+If the work defined in a Working Group's charter is complete, the charter
+should be revoked.
+
+A Working Group's charter can be revoked either by consent of the Working
+Group's members or by a CPC vote. Once revoked, any future work that arises
+becomes the responsibility of the CPC.
+
+## Joining a WG
+
+To find out how to join a working group, consult the GOVERNANCE.md in
+the working group's repository.
+
+## Starting A Core Working Group
+
+A Working Group is established by first defining a charter that can be
+ratified by the CPC. A charter is a *statement of purpose*, a
+*list of responsibilities* and a *list of initial membership*.
+
+A working group needs 3 initial members. These should be individuals
+already undertaking the work described in the charter.
```

Comment:
Does the CPC need a designated point of contact?

Does /cross-project-council contain a list of working groups with points of contact?

Is that the (currently empty) section at the bottom?

Reply:
We should likely maintain a table of working groups & and their POCs, I agree. I think designating the 'current working groups' section at the bottom of this document sounds good for now, until we create a higher level doc that we can migrate our (maintained) table to – where it's more accessible for the public (eg. providing an 'Introduction to OpenJS Working Groups' document, rather than pointing folks to our admin policy doc).

Reply:
Yes, as you'll see in the Node.js version of this doc, the Current Working groups is fleshed out: https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md#current-working-groups